src/OpenColorIO/Baker.cpp

```diff
@@ -275,10 +275,10 @@ OCIO_NAMESPACE_EXIT
 #ifdef OCIO_UNIT_TEST
 
 namespace OCIO = OCIO_NAMESPACE;
-#include "unittest.h"
+#include "UnitTest.h"
 
 /*
-OIIO_ADD_TEST(Baker_Unit_Tests, test_listlutwriters)
+OCIO_ADD_TEST(Baker_Unit_Tests, test_listlutwriters)
```

Comment:
Not only is this ok with me, but I continue to be mystified how it got this name in the first place, since OIIO never had an ADD_TEST!